website/addons/s3/static/user-cfg.js

```diff
@@ -1,61 +1,11 @@
-'use strict';
+var $osf = require('js/osfHelpers');
+var S3ViewModel = require('./s3UserConfig.js').S3ViewModel;
 
-var $ = require('jquery');
-var bootbox = require('bootbox');
-var AddonHelper = require('js/addonHelper');
-var language = require('js/osfLanguage').Addons.s3;
+// Endpoint for S3 user settings
+var url = '/api/v1/settings/s3/accounts/';
 
-$('#s3RemoveAccess').on('click', function() {
-    bootbox.confirm({
-        title: 'Disconnect S3 Account?',
-        message: language.confirmDeauth,
-        callback: function(result) {
-            if(result) {
-                deleteToken();
-            }
-        },
-        buttons:{
-            confirm:{
-                label:'Disconnect',
-                className:'btn-danger'
-            }
-        }
-    });
-});
+var s3ViewModel = new S3ViewModel(url);
+$osf.applyBindings(s3ViewModel, '#s3AddonScope');
```

Comment:
Why not initialize `S3UserConfig`, which instantiates `S3ViewModel` and does `applyBindings`?